### PR TITLE
Linter_Bears_Advanced.rst: Improved the message for config file

### PR DESCRIPTION
**Pull request by yashLadha**
Message Note has been improved for the config file generation, which
may yields better performance.

Closes Issue: https://github.com/coala/coala/issues/4119

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [*] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [*] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
Hi @yashLadha , in your commit message "Closes Issue: ..." should be "Closes ..."

**Comment by yashLadha** [comment]
Done.

**Comment by yukiisbored** [comment]
Hello @yashLadha, You don't need `:` in `Closes`

**Comment by yashLadha** [comment]
Does it makes any difference?

File: docs/Developers/Linter_Bears_Advanced.rst [hunk 1]
```diff
@@ -43,7 +43,10 @@ The path of the temporary configuration file can be accessed inside
 
 .. note::
 
-    By default, no configuration file is generated.
+   By default, no configuration file is generated, as all the necessary
```
Comment (Makman2) [inline docs/Developers/Linter_Bears_Advanced.rst]: 4 spaces indentation :+1:

File: docs/Developers/Linter_Bears_Advanced.rst [hunk 2]
```diff
@@ -43,7 +43,10 @@ The path of the temporary configuration file can be accessed inside
...
+   settings are already bundled with the linter. But if the user wants
+   use some external tools which may increase the performance of the
+   linter, them they can ask for a config file to generate.
```
Comment (Makman2) [inline docs/Developers/Linter_Bears_Advanced.rst]: actually that's wrong and that's not what I meant ;) You should read the code of `JSHintBear` I've given you :+1:

**Comment by jayvdb** [comment]
Ping @yashLadha, this is really important as our dev docs needs to describe how to write bears so they perform well.

**Pull request closed without merging**